src/server/common/helpers/build-error-details.js

```diff
@@ -5,10 +5,24 @@ function buildErrorMessages(path, value) {
   return object
 }
 
+function extractMessage(detail) {
```

Comment:
I am struggling to get `alternative` error messages. Locally I am just returning `detail.message` from thie method and its all working as expected. I'm not sure if the`alternatives.match` issue this work is fixing is still actually a problem

Reply:
think removing the .maxSize/mime and moving it server side resolved it?